test/MockRelay.cpp

```diff
@@ -11,6 +11,7 @@
 #include "MockRelay.h"
 
 bool dual_tor_sock = false;
+char loopback[IF_NAMESIZE] = "Loopback0";
```

Comment:
Can we add corresponding unittest for dualtor behavior (server callback dualtor and relay client socket)?

Reply:
Sure, will update in latest diff.